compiler/one-cmds/onelib/constant.py

```diff
@@ -29,6 +29,7 @@ class CONSTANT:
         'fold_dwconv',
         'fold_fully_connected',
         'fold_gather',
+        'fold_shape',
```

Comment:
Isn't it necessary to add this option into `OPTIMIZATION_OPTION`(line 83) either?

Reply:
(oops, duplicated)